lib/api/pokedex_api_spec/handler/pokemon_handler.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:pokedex/api/pokedex_api_spec/api_client.dart';
+import 'package:pokedex/api/pokedex_api_spec/model/pokemon_model.dart';
+
+class PokemonApi {
+  final ApiClient apiClient;
+
+  PokemonApi(this.apiClient);
+
+  static const int _limitAmount = 20;
+
+  Future<Pokemon> getPokemonList() async {
```

Comment:
this is going to be refactored when you implement pagination. you decide, when do you refactor it? now or when pagination is being implemented?

Reply:
should I refactor it to like getInitialList?

Reply:
no. getPokemonList should accept value of offset and limit

Reply:
let's do this next time @yohancai no need to do it now.